src/plugins/es_ui_shared/static/forms/hook_form_lib/hooks/use_form.ts

```diff
@@ -659,6 +659,15 @@ export function useForm<T extends FormData = FormData, I extends FormData = T>(
   // ----------------------------------
   // -- EFFECTS
   // ----------------------------------
+  useEffect(() => {
```

Comment:
As mentioned in my previous comment, please don't put the `useEffect` back. We can discuss over Zoom how to solve your form without counting on this.